parallel_test.go

```diff
@@ -111,7 +110,10 @@ func TestParallelContainers(t *testing.T) {
 			}
 
 			for _, c := range res {
-				defer c.Terminate(context.Background())
+				c := c
```

Comment:
I guess this line comes because of https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable

Reply:
that's right, without we'll reference first `c` each execution